pkg/commands/edit/add/flagsandargs.go

```diff
@@ -54,10 +55,37 @@ func (a *flagsAndArgs) Validate(args []string) error {
 }
 
 func (a *flagsAndArgs) ExpandFileSource(fSys fs.FileSystem) error {
```

Comment:
I know this function has no comment, but it needs one.
Suggest something like

```
// ExpandFileSource normalizes a string list, possibly
// containing globs, into a validated, globless list.
// For example, this list:
//     some/path
//     some/dir/a*
//     bfile=some/dir/b*
// becomes:
//     some/path
//     some/dir/airplane
//     some/dir/ant
//     some/dir/apple
//     bfile=some/dir/banana
// i.e. everything is converted to a key=value pair,
// where the value is always a relative file path,
// and the key, if missing, is the same as the value.
// In the case where the key is explicitly declared,
// the globbing, if present, must have exactly one match.
```

Reply:
Quite robust! Using as is. Thank you!